src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -207,6 +207,43 @@ indexvector(P::SPZ) = P.idx
 Returns a collection of n unique identifiers (intergers 1, …, n).
 """
 uniqueID(n::Int) = 1:n
+
+"""
+    mergeID(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+Returns two polynomial zonotopes equivalent to the correspondent input but with the same
+index vector.
+
+### Input
+
+- `P1` -- sparse polynomial zonotope
+- `P2` -- sprase polynomial zonotope
+
+### Output
+
+two polynomial zonotopes with the same index vector
+"""
+function mergeID(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+    idx1 = indexvector(P1)
+    idx2 = indexvector(P2)
+    idx1 == idx2 && return (P1, P2)
+
+    H = setdiff(idx2, idx1)
+    idx_new = vcat(idx1, H)
```

Comment:
I am not sure this is correct. It seems non-symmetric. Does the test still work in the other order?

Reply:
I think it's symmetric up to permutation. That is the merged zonotopes will have the same parameters, but these will not necessarily ordered by index (i.e. the index vector is not ordered). If one also sorted the index vector, then it would be symmetric I think

Reply:
at least looking at the algorithm description on page 35 of Niklas' thesis, the algorithm doesn't seem to be symmetric (because it doesn't fix the order)

Reply:
So when you do `mergeID(PZ2, PZ1)` in the test case, you get the IDs `[2, 4, 1, 3]`. Is this acceptable or should the indices be sorted?

Reply:
based on today's conversation, should we normalize the order?

Reply:
if in functions that require the same idvector (quadratic map, exact sum) you so something like

```julia
indexvector(PZ1) == indexvector(PZ2) || (PZ1, PZ2 = mergeID(PZ1, PZ2)
```

I think even without normalizing the order it would be correct, because calling `mergeID` would not change `PZ1, PZ2`, on the other hand, I don't see harm in normalizing the order either

Reply:
I would not be surprised if some operations assume the same order after `mergeID`. So I would sort.